docs/usermanual/index.md

```diff
@@ -17,16 +17,156 @@ TODO
 
 ## AWS
 
-TODO
-
 ### Disk snapshot
 
-TODO
+Use case: you have a volume `vol1` in an AWS account (Account A, in zone `us-east-2b`) which you would like to make a copy 
+of in a different account (Account B).
+
+Your AWS credentials configuration (`~/.aws/credentials`) should look like this:
+
+```text
+# Access credentials to Account A
+[src_profile]
+aws_access_key_id = xxx
+aws_secret_access_key = xxx
+
+# Access credentials to Account B
+[dst_profile]
+aws_access_key_id = yyy
+aws_secret_access_key = yyy
+
+# Access credentials to the default account to use with AWS operations. Note that this can be the same account as one 
+# of the accounts above. This is the account that will be used by libcloudforensics for AWSAccount objects that are not 
+# instantiated with a particular profile.
+[default]
+aws_access_key_id = zzz
+aws_secret_access_key = zzz
+```
+
+#### Using the library
+
+To make the copy, import the `forensics` package and use the `CreateVolumeCopy` method.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another 
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='analysis_profile')
+```
+
+If you want to make the copy in a different zone, just specify `dst_zone` in the above method.
+You can also pass a dictionary of tags to add to the volume copy.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the volume 'vol1' from one account to another, in a different zone
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  dst_zone='us-west-2a',
+                                  volume_id='vol1', 
+                                  src_profile='src_profile', 
+                                  dst_profile='analysis_profile',
+                                  tags={'customTag': 'customValue'})
+```
+
+Finally, you can opt to pass an `instance_id` parameter to the method instead of a `volume_id`.
+If you do so, then the boot volume of the instance pointed to by `instance_id` will be copied.
+
+```python
+from libcloudforensics.providers.aws import forensics
+
+# Create a forensic copy of the boot volume of instance 'inst1' in the default account.
+
+copy = forensics.CreateVolumeCopy('us-east-2b',
+                                  instance_id='inst1')
+```
+
+#### Using the CLI
+
+All snippets of code above have their CLI equivalent:
```

Comment:
can we add each cli snippet below the python example ? it is easier to follow for the reader.